src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -860,6 +872,184 @@ void unpack_u4f16(const ov::SoPtr<ov::ITensor>& from,
     }
 }
 
+void unpack_u4f16_asymm_zp(const ov::SoPtr<ov::ITensor>& from,
```

Comment:
can we add a tests for Alexey PR, or to a branch based on his PR or  at least notice in TODO for after work required. 
https://github.com/openvinotoolkit/openvino/pull/25780

Reply:
Will add it asap.